templates/credentialing/index.html

```diff
@@ -0,0 +1,149 @@
+{% extends "credentialing/base_cube.html" %}
+
+{% block title %}Canonical Certification{% endblock %}
+
+{% block meta_description %}The Canonical Ubuntu Essentials exams certify knowledge and verify skills in general Linux, Ubuntu Desktop, and Ubuntu Server topics.{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1QuhO-9FEOGLrYp8bErS_9snqdljl7d6tFAUoNQxoVDQ/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>Ubuntu Linux Certifications</h1>
+      <p class="p-heading--4">Stand out! Earn the official Canonical Ubuntu badge for FREE.</p>
```

Comment:
@mrgnr @linux-in-flux Nitpick, I'm not sure, but I don't think this is 'Canonical style' (to capitalise the full word) I would suggest changing it to "free" rather than "FREE".